extras/lbf-nix/lbf-rust.nix

```diff
@@ -86,7 +86,7 @@ let
     {
       num-bigint = "~0.4";
       serde_json = { version = "^1.0"; features = [ "arbitrary_precision" ]; };
-      plutus-ledger-api = { git = "https://github.com/mlabs-haskell/plutus-ledger-api-rust"; features = [ "lbf" ]; rev = "d66d39c949f59e543c91ee36377c93422d8e9d75"; };
+      plutus-ledger-api = { path = "../plutus-ledger-api-0.1.0"; features = [ "lbf" ]; };
```

Comment:
We're using a flat dependency structure in extras, so for instance if lbr-prelude depends on lbr-prelude-derive, then they'll be all in the same .extras directory (no .extras inside lbr-prelude). This means, that if a package is inside the `.extras` directory, it has to go up: `../`.
This version configuration table is used to generate Cargo.toml for the autogenerated code, so it will be always inside `.extras`.
For other packages, we had the `vendoredSrc` in rust flake, which will change paths from `.extras` to `../`

In an ideal future version, I want to use a local cargo registry instead of direct paths, that could improve the ergonomy of pkg management.

Reply:
This is very implicit and depends very much on the context of how it is used. Slap a comment explaining this because it will break us am sure.

The local cargo registry sound ideal! Basically that is the correct solution throughout! We'll hit that in all other places as well.